.devcontainer/devcontainer.json

```diff
@@ -12,7 +12,7 @@
   },
   // Port 5683 udp is used by Shelly integration
   "appPort": ["8123:8123", "5683:5683/udp"],
-  "runArgs": ["-e", "GIT_EDITOR=code --wait"],
+  "runArgs": ["-e", "GIT_EDITOR=code --wait", "--network=host"],
```

Comment:
Host network should not be used here, as that isn't available for everyone.

Reply:
just retested without and can also reach from browser the UI on the mapped port so not required,
although one difference is that with "host" vscode will provide an handy popup with a link to open in browser the instant the port becomes available, a bummer not to have it 😞

https://github.com/user-attachments/assets/abbca2b4-b742-43e6-a85d-6f6da7ac1e88

Reply:
Cool, but we can't use it; as not all systems support it.